storage_service/locations/models/package.py

```diff
@@ -2062,25 +2062,25 @@ def get_fixity_check_report_send_signals(
         # Build the response (to be a JSON object)
         response = {
             "success": success,
-            "message": message,
+            "message": str(message),
             "failures": {"files": {"missing": [], "changed": [], "untracked": []}},
             "timestamp": timestamp,
         }
         for failure in failures:
             if isinstance(failure, bagit.FileMissing):
-                info = {"path": failure.path, "message": str(failure)}
+                info = {"path": failure.path, "message": failure}
```

Comment:
Why do we have to remove the `str` here and below?

Reply:
oh yes that's bagit exceptions.